e2e/fragments/LabelStudio.js

```diff
@@ -4,8 +4,14 @@ const { I } = inject();
 const Helpers = require("../tests/helpers");
 
 module.exports = {
-  init(params) {
-    I.executeScript(Helpers.initLabelStudio, params);
+  init({ events = {}, ...params }) {
```

Comment:
Maybe add a comment here about the difference about the `events` binding vs the dynamic function injections we are using at runtime.